apps/react-app/src/pods/manfred-json-input/manfred-json-input.styles.ts

```diff
@@ -1,8 +1,115 @@
 import { css } from '@emotion/css';
 import { theme } from '@/core/theme';
 
+export const nav = css`
+  display: flex;
+  position: fixed;
+  top: 0;
+  flex-direction: row;
+  justify: space-between;
+  height: 50px;
+  width: 100%;
+  left: 0px;
+  top: 0px;
+  border-radius: 0px;
+  align-items: center;
+  padding: 8px 16px;
+  gap: 10px;
+  background: ${theme.palette.dark[900]};
+`;
+
+export const container = css`
+  width: 1024px;
+`;
+
+export const divh1 = css`
+  @media (min-width: 1280px) {
+    margin: 0 auto;
+    width: 75%;
+  }
+`;
+
 export const header = css`
-  color: ${theme.palette.primary[500]};
+  color: ${theme.palette.primary[50]};
   font: ${theme.typography.desktop.h1};
   padding: ${theme.spacing(4)};
+
+  & .span1 {
+    color: ${theme.palette.primary[500]};
+  }
+
+  & .span2 {
+    color: ${theme.palette.secondary[500]};
+  }
+
+  @media (min-width: 390px) and (max-width: 843px) {
+    text-align: center;
+    display: block;
+    margin: 0 auto;
+    font: ${theme.typography.desktop.h3};
+    padding: 0;
+  }
+
+  @media (min-width: 834px) and (max-width: 1280px) {
+    text-align: center;
+    display: block;
+    margin: 0 auto;
+    font: ${theme.typography.desktop.h2};
+  }
+
+  @media (min-width: 1280px) {
+    text-align: center;
+    display: block;
```

Comment:
you have duplicated of these props in all @media queries, think about this and put in correct place